Cargo.toml

```diff
@@ -1,3 +1,11 @@
 [workspace]
 members = ["crates/*"]
 resolver = "2"
+
+[workspace.package]
+license = "MIT OR Apache-2.0"
+repository = "https://github.com/rust-cli/anstyle.git"
+homepage = "https://github.com/rust-cli/anstyle"
+categories = ["command-line-interface"]
+keywords = ["ansi", "terminal"]
```

Comment:
I intentionally kept these out as we are likely to have deviations on these from crate to crate and want to re-evaluated for each

Reply:
Yeah i'm a dope, forgot to pull your changes. Let me get those fixed.